components/director/pkg/graphql/integrationsys_validation.go

```diff
@@ -0,0 +1,13 @@
+package graphql
+
+import (
+	validation "github.com/go-ozzo/ozzo-validation"
+	"github.com/kyma-incubator/compass/components/director/pkg/inputvalidation"
+)
+
+func (i IntegrationSystemInput) Validate() error {
+	return validation.ValidateStruct(&i,
+		validation.Field(&i.Name, validation.Required, inputvalidation.Name),
+		validation.Field(&i.Description, validation.NilOrNotEmpty, validation.Length(1, 128)),
```

Comment:
We decided that optional text fields can now be empty, so `NilOrNotEmpty` rule should be deleted from them.